l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -195,51 +195,13 @@ def check_partner_base_data(self, partner_id, DatiAnagrafici):
                 % (DatiAnagrafici.Anagrafica.Cognome, partner.lastname)
             )
 
-    def getPartnerBase(self, DatiAnagrafici):  # noqa: C901
+    def getPartnerBase(self, DatiAnagrafici, partnerCreate=True):  # noqa: C901
```

Comment:
Ora che il metodo è stato semplificato forse si può riattivare il linting
```suggestion
    def getPartnerBase(self, DatiAnagrafici, partnerCreate=True):
```

Reply:
:+1: